src/helpers/mergeContact.spec.js

```diff
@@ -0,0 +1,75 @@
+import { mergeContact } from './mergeContact'
+
+describe('Merge contacts', () => {
+  test('should merge properties', () => {
+    const fakeContact = {
+      email: [{ address: 'starck@localhost' }],
+      name: { familyName: 'Starck' },
+      phone: [{ number: '0987654321', primary: true, type: 'Work' }],
+      address: [{ formattedAddress: 'Westeros', primary: true, type: 'Work' }]
+    }
+    const importedContact = {
+      name: { familyName: 'Starck', givenName: 'Ned' },
+      phone: [{ number: '0123456789' }],
+      email: [{ address: 'ned@localhost' }]
```

Comment:
In this test, we are given two contacts and check the merge with the expected merged contact. We don't decide if the contact should be merged or not. But you are right, we could add a property common to both contacts, but it will not change the test expectation.